File: eRakija/Models/Proizvod.cs
namespace eRakija.Models
{
    public class Proizvod
    {
    [Key]
    public int ID { get; set; }
    public required string Naziv { get; set; }
    public required int Cena { get; set; }

    public string? Slika { get; set; }
    public string? Opis { get; set; } 

    public int? Kolicina { get; set; }
    public TipProizvoda? tipProizvoda { get; set; }
    }
}

File: eRakijaFR/src/app/models/Proizvod.ts
import { TipProizvoda } from "./TipProizvoda"

export interface Proizvod{
    id: number
    naziv: string
    cena: number
    slika: string
    opis: string
    kolicina: number
    tip: TipProizvoda
}

export interface ProizvodPost{
    naziv: string
    cena: number
    slika: string
    opis: string
    kolicina: number
}